skycultures/bali/index.json

```diff
@@ -0,0 +1,859 @@
+        27654,
+        25985,
+        24305
+      ]],
+	  "image": {
+		  "file": "illustrations/Bade.png",
+		  "size": [1500, 1120],
+		  "anchors": [
+		  {"pos": [495, 1010], "hip": 23685},
+		  {"pos": [790, 271], "hip": 27654},
+		  {"pos": [953, 1010], "hip": 24305}]
+	  },
+      "common_name": {
+        "english": "Funeral Bier",
+        "native": "ᬩᬤᬾ",
+        "pronounce": "Bade",
+        "IPA": "/ba.de/"
+      }
+    },
+    {
+      "id": "CON bali 028",
+      "lines": [[
+        93747,
+        95501,
+        99473,
+        97278,
+        93747],[
+        97649,
+        93805
+      ]],
+	  "image": {
+		  "file": "illustrations/Ru.png",
+		  "size": [1335, 1575],
+		  "anchors": [
+		  {"pos": [247, 582], "hip": 93747},
+		  {"pos": [602, 818], "hip": 97278},
+		  {"pos": [1124, 729], "hip": 99473}]
+	  },
+      "common_name": {
+        "english": "Arrow",
+        "native": "ᬭᬸ",
+        "pronounce": "Ru",
+        "IPA": "/ˈru/"
+      }
+    },
+    {
+      "id": "CON bali 029",
+      "lines": [[
+        100453,
+        91262,
+        87833,
+        100453
+      ]],
+	  "image": {
+		  "file": "illustrations/Magelut.png",
+		  "size": [1200, 766],
+		  "anchors": [
+		  {"pos": [520, 156], "hip": 91262},
+		  {"pos": [1008, 613], "hip": 100453},
+		  {"pos": [165, 478], "hip": 87833}]
+	  },
+      "common_name": {
+        "english": "Quarrel over Debt",
+        "native": "ᬫᬕᭂᬮᬸᬢ᭄",
+        "pronounce": "Magělut",
+        "IPA": "/magəlut/"
+      }
+    },
+    {
+      "id": "CON bali 030",
+      "lines": [[
+        85258,
+        74395,
+        78384
+      ]],
+	  "image": {
+		  "file": "illustrations/Banyak_Angrem.png",
+		  "size": [1361, 1471],
+		  "anchors": [
+		  {"pos": [818, 242], "hip": 85258},
+		  {"pos": [366, 970], "hip": 74395},
+		  {"pos": [1002, 1138], "hip": 78384}]
+	  },
+      "common_name": {
+        "english": "The Brooding Goose",
+        "native": "ᬩᬜᬓ᭄ᬳᬗᬺᬫ᭄",
+        "pronounce": "Banyak Angrěm",
+        "IPA": "/ˈbaɲaʔ.aŋrəm/"
+      }
+    },
+    {
+      "id": "CON bali 031",
+      "lines": [[
+        62894,
+        61585,
+        68191
+      ]],
+	  "image": {
+		  "file": "illustrations/Bubu_Bolong.png",
+		  "size": [2048, 1981],
+		  "anchors": [
+		  {"pos": [1485, 1206], "hip": 62894},
+		  {"pos": [525, 1107], "hip": 61585},
+		  {"pos": [1260, 372], "hip": 68191}]
+	  },
+      "common_name": {
+        "english": "Leaky Fish Trap",
+        "native": "ᬩᬸᬩᬸᬩᭀᬮᭀᬂ",
+        "pronounce": "Bubu Bolong",
+        "IPA": "/bubu.ˈbolɔŋ/"
+      }
+    }
+  ],
+  "common_names": {
+    "NAME Sun": [
+      {
+        "english": "The Sun",
+        "native": "ᬲᬸᬗᬾᬗᬾ",
+        "pronounce": "Sungenge",
+		"IPA": "/su.ŋɛ.ŋɛ/",
+        "references": "[1,2]"
+      }
+    ],
+    "coalsack nebula": [
+      {
+        "english": "Leaky Fish Trap",
+        "native": "ᬩᬸᬩᬸᬩᭀᬮᭀᬂ",
+        "pronounce": "Bubu bolong",
+		"IPA": "/bubu.ˈbolɔŋ/"
```

Comment:
This line results in invalid JSON. Also, the file is now inconsistent regarding spaces vs tabs indentation.